uint64 timestamp
uint8 frame
uint8 count
uint8[2] _padding0

uint8 MAX_OBSTACLES = 4
uint8 FRAME_LOCAL_NED = 1
uint8 FRAME_LOCAL_ENU = 4

float32[4] x
float32[4] y
float32[4] z
float32[4] radius
float32[4] margin
